packages/features/ee/payments/api/webhook.ts

```diff
@@ -315,6 +315,8 @@ const handleSetupSuccess = async (event: Stripe.Event) => {
     bookingData.status = BookingStatus.ACCEPTED;
   }
 
+  if (booking.location) evt.location = booking.location;
```

Comment:
I think this could be moved to `getBooking` itself which creates evt ? Also as a follow-up maybe we should think about making the `location` a required property in `CalendarEvent` type